x/erc20/types/contracts/ERC20DirectBalanceManipulation.sol

```diff
@@ -2,7 +2,7 @@
 
 pragma solidity ^0.8.0;
 
-import "@openzeppelin/contracts/token/ERC20/presets/ERC20PresetMinterPauser.sol";
+import "./@openzeppelin/contracts/token/ERC20/presets/ERC20PresetMinterPauser.sol";
```

Comment:
We can't use the npm package format?

Reply:
@fedekunze the npm import doesn't work locally when I try to compile the contracts. I decided to do it this way, as I think vscode requires a specific setup: https://forum.openzeppelin.com/t/source-openzeppelin-not-found-file-import-callback-not-supported/1812

Reply:
I moved the contracts directory to top-level though!